src/js/components/reviewData/ReviewDataNarrative.jsx

```diff
@@ -7,6 +7,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReviewDataNarrativeDropdown from './ReviewDataNarrativeDropdown';
 import ReviewDataNarrativeTextfield from './ReviewDataNarrativeTextfield';
+import { createOnKeyDownHandler } from '../../helpers/util';
+import * as Icons from '../SharedComponents/icons/Icons';
```

Comment:
Alternatively, just import the icon we need
`import { CloudDownload } from '../SharedComponents/icons/Icons';`